### PR TITLE
Terminal crash fix

### PR DESCRIPTION
**Pull request by philvukovic**
* use `bash` everywhere
* move `update_spimbot.sh` to `.bashrc`
* fix exiting on error issue (`set -e` removed)
* added QtSpimbot in `~/bin` so that `QtSpimbot ...` works
* add auto image pull to run command

### CODE REVIEW

File: image/home/.bashrc [hunk 1]
```diff
@@ -22,17 +20,19 @@ if [ ! -e "/home/ubuntu/.spimbot/spimbot-binaries" ]; then
...
 fi;
 
 cp /home/ubuntu/.spimbot/spimbot-binaries/$BINARY_VERSION/QtSpimbot $DEST_PATH
-echo "Installed new binary"
-cd ~/shared
+cp /home/ubuntu/.spimbot/spimbot-binaries/$BINARY_VERSION/QtSpimbot /home/ubuntu/bin
+echo "QtSpimbot updated"
+cd /home/ubuntu/shared
+
+export PATH=$PATH:"/home/ubuntu/bin"
```
Comment (Nydauron) [inline image/home/.bashrc]: This line can be moved to here
https://github.com/cs233/ubuntu-vnc-desktop-spimbot/blob/a04ef98ef76cbd734aee821bf4dcaee58f773c1a/image/home/.bashrc#L12-L14

File: image/home/.bashrc
Comment (Nydauron) [inline image/home/.bashrc:1]: Can we copy over the .bashrc currently in the container (the container has a default .bashrc for ubuntu) and put this into another file `.update_spimbot.sh` then add `source .update_spimbot.sh` to the `.bashrc` file?

**Review by Nydauron — CHANGES_REQUESTED** [review]

File: image/home/.bashrc
Comment (philvukovic) [reply image/home/.bashrc:1]: Yeah, good idea

File: image/home/.bashrc (hunk 1 above)
Comment (philvukovic) [reply image/home/.bashrc]: agree

File: image/home/.bashrc
Comment (philvukovic) [reply image/home/.bashrc:1]: might require creating home directory explicitly though, needs a little testing

Comment (Nydauron) [reply image/home/.bashrc:1]: > might require creating home directory explicitly though

This is probably not necessary. There already exists a `.bashrc` in the container

File: image/home/.bashrc (hunk 1 above)
Comment (philvukovic) [reply image/home/.bashrc]: fixed in latest

File: image/home/.bashrc
Comment (philvukovic) [reply image/home/.bashrc:1]: changed to `echo [...] >> .bashrc` in Dockerfile

**Review by Nydauron — APPROVED** [review]

**Pull request merged**